src/core/platform.js

```diff
@@ -9,6 +9,16 @@ let gamepads = false;
 let workers = false;
 let passiveEvents = false;
 
+// If the environment doesn't support the Gamepad API, we return the same array (GC friendly).
+// Examples: node or browsers that hide USB devices to prevent fingerprinting of users.
+const dummy = [];
+const getDummy = () => dummy;
+
+/**
+ * @type {typeof navigator.getGamepads}
```

Comment:
JSDocs is not keen on `typeof` - I tried to use it few times already. Any workaround?